src/bintools/dwarf/expressions.py

```diff
@@ -113,7 +113,7 @@ def __init__(self, dwarf, length):
             opcode = dwarf.u08()
             if opcode not in DW_OP:
                 previnst = ','.join(str(x) for x in self.instructions)
-                raise ParseError("Unknown DW_OP code: %d (after %s, offset 0x%x)" % (opcode, previnst, dwarf.io.tell()))
+                break #raise ParseError("Unknown DW_OP code: %d (after %s, offset 0x%x)" % (opcode, previnst, dwarf.io.tell()))
```

Comment:
It's been a long time and I'm not sure what these expressions are exactly used for, but I don't think simply ignoring errors is the right way to fix this?